dev/NavigationView/NavigationView.cpp

```diff
@@ -1398,6 +1434,88 @@ void NavigationView::UpdateAdaptiveLayout(double width, bool forceSetDisplayMode
     }
 }
 
+void NavigationView::UpdatePaneLayout()
+{
+    if (!IsTopNavigationView())
+    {
+        auto availableHeight = 0.0;
+        if (const auto &paneContentRow = m_itemsContainerRow.get())
+        {
+            // 20px is the padding between the two item lists
+            availableHeight = paneContentRow.ActualHeight() - 28;
+        }
+        if (const auto &paneFooter = m_leftNavFooterContentBorder.get())
+        {
+            availableHeight -= paneFooter.ActualHeight();
+        }
+
+        // Only continue if we have a positive amount of space to manage.
+        if (availableHeight > 0)
+        {
+            // We need this value more than twice, so cache it.
+            const auto availableHalf = availableHeight / 2;
+
+            // Footer items should have precedence as that usually contains very
+            // important items such as settings or the profile.
+            if (const auto& footerItemsScrollViewer = m_footerItemsScrollViewer.get())
+            {
+                if (const auto& footerItemsRepeater = m_leftNavFooterMenuRepeater.get())
+                {
+                    const auto height = footerItemsRepeater.ActualHeight();
+                    const auto desired = footerItemsRepeater.DesiredSize().Height;
+
+                    // We know the actual height of footer items, so use that to determine how to split pane.
+                    if (const auto& menuItems = m_leftNavRepeater.get())
+                    {
+                        const auto footersActualHeight = footerItemsRepeater.ActualHeight();
```

Comment:
>footerItemsRepeater [](start = 57, length = 19)

Isn't this the height local variable from the scope above?

Reply:
The height and desired height from above were for debugging and are unused. I'll remove them.